config/software/datadog-trace-agent.rb

```diff
@@ -0,0 +1,35 @@
+name "datadog-trace-agent"
+default_version "master"
+
+gourl = "https://storage.googleapis.com/golang/go1.7.1.linux-amd64.tar.gz"
+goout = "go.tar.gz"
+godir = "/usr/local/go17"
+gobin = "#{godir}/go/bin/go"
+
+env = {
+  "GOPATH" => "#{Omnibus::Config.cache_dir}/src/#{name}",
+  "GOROOT" => "/usr/local/go17/go",
+  "PATH" => "#{godir}/go/bin:#{ENV["PATH"]}"
+}
+
+build do
+   ship_license "https://raw.githubusercontent.com/DataDog/datadog-trace-agent/#{version}/LICENSE"
+   # download go1.7
+   command "curl #{gourl} -o #{goout}"
+   command "mkdir -p #{godir}"
+   command "tar zxfv #{goout} -C #{godir}"
+
+   # Download datadog-trace-agent without installing
+   command "git clone https://github.com/DataDog/datadog-trace-agent.git $GOPATH/src/github.com/DataDog/datadog-trace-agent", :env => env
```

Comment:
you shouldn't clone that manually but use the `source` helper.

e.g.:
https://github.com/DataDog/dd-agent-omnibus/blob/master/config/projects/datadog-agent.rb#L23

It's way cleaner and easier to get the right version